src/main/java/util/FormplayerDateUtils.java

```diff
@@ -0,0 +1,28 @@
+package util;
+
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+import java.util.TimeZone;
+
+/**
+ * Date utility functions for dealing with dates in formplayer
+ */
+public class FormplayerDateUtils {
+    public static String convertJavaDateStringToISO(String date) {
+        DateFormat dfFrom = new SimpleDateFormat("EEE MMM dd kk:mm:ss z yyyy", Locale.ENGLISH);
+        Date result;
+        try {
+            result =  dfFrom.parse(date);
+        } catch (ParseException e) {
+            // Could not parse date
+            return null;
+        }
+        TimeZone tz = TimeZone.getTimeZone("UTC");
```

Comment:
I believe that should be the timezone we're converting to, see test for confirmation: https://github.com/dimagi/formplayer/pull/154/files/039fc5ec595295f9e1f223f2144df1fc75f06bb1#diff-d98abeb04d3db7c31c3f68b399f19511R21